flink-runtime/src/test/java/org/apache/flink/runtime/blob/BlobCacheRetriesTest.java

```diff
@@ -148,7 +149,7 @@ void testBlobForJobFetchWithTooManyFailuresHa() throws IOException {
             testBlobFetchWithTooManyFailures(blobStoreService, new JobID(), PERMANENT_BLOB);
         } finally {
             if (blobStoreService != null) {
-                blobStoreService.closeAndCleanupAllData();
+                blobStoreService.cleanupAllData();
```

Comment:
```suggestion
               blobStoreService.cleanupAllData();
               blobStoreService.close();
```

Reply:
There seems to be one step missing here, right?

Reply:
Thanks @gtk96 for catching this.
Could you help review it again when you have time?